WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -171,6 +178,15 @@ final class OrdersRootViewController: UIViewController {
         setupNavigation(viewModel: viewModel)
     }
 
+    /// Attempts to navigate to the first Order in the Order List by opening its details
+    ///
+    private func navigateToFirstOrderIfPossible() {
```

Comment:
Suggestion: If we do this navigation only for iPad, 
1. What do you think about moving the check `UIDevice.current.userInterfaceIdiom == .pad` to this method? This way, we will only do the "first order navigation" on iPad. (Prevents this navigation from happening in iPhone)
2. We can also rename the method to clarify that this only applies to iPad.

Reply:
Generally I prefer to not call the method at all rather than calling it and then returning earlier, but is also true that this way the iPad logic is encapsulated in the method rather than relying on a check somewhere else. Changed on https://github.com/woocommerce/woocommerce-ios/pull/11571/commits/1a2b5b6cc9f6231048a03d00b31185ea6ff82d7f